### PR TITLE
linux-intel-rt-acrn-uos: add v5.4 recipe

### PR DESCRIPTION
**Pull request by cheeyanglee**
add v5.4 linux-intel-rt-acrn-uos kernel and set it as default
UOS rt kernel version.

Signed-off-by: Chee Yang Lee <chee.yang.lee@intel.com>

### CODE REVIEW

File: recipes-kernel/linux/linux-intel-rt-acrn-uos_5.4.bb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+SUMMARY = "Linux Preempt RT Kernel with ACRN enabled (UOS)"
```
Comment (saininav) [inline recipes-kernel/linux/linux-intel-rt-acrn-uos_5.4.bb:1]: Can we split it into .inc and .bb ? 
*.inc can also be included in 4.19 RT recipe as well. So we can avoid code duplication (in dunfell).

Comment (cheeyanglee) [reply recipes-kernel/linux/linux-intel-rt-acrn-uos_5.4.bb:1]: for reduce code duplication purpose, it would need to refactor other kernel recipe too

Comment (saininav) [reply recipes-kernel/linux/linux-intel-rt-acrn-uos_5.4.bb:1]: If you mean 4.19 RT recipe only,  then please do changes.

Comment (cheeyanglee) [reply recipes-kernel/linux/linux-intel-rt-acrn-uos_5.4.bb:1]: there are other duplicates as well, see line for KERNEL_FEATURES and DEPENDS

**Pull request merged**